composer.json

```diff
@@ -21,13 +21,13 @@
     }
   ],
   "require": {
-    "php": ">=5.2",
+    "php": ">=7.3",
```

Comment:
I think this would be considered a backwards incompatible change, so we should bump the version up to 2.0.0